2023-10-15 20:32:29.171 PKT [3412] LOG:  checkpoint starting: time
2023-10-15 20:32:30.094 PKT [3412] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.877 s, sync=0.021 s, total=0.923 s; sync files=9, longest=0.013 s, average=0.003 s; distance=10 kB, estimate=10 kB; lsn=0/6DB1798, redo lsn=0/6DB1760
2023-10-15 20:47:29.113 PKT [3412] LOG:  checkpoint starting: time
2023-10-15 20:47:30.003 PKT [3412] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.859 s, sync=0.023 s, total=0.891 s; sync files=9, longest=0.014 s, average=0.003 s; distance=11 kB, estimate=11 kB; lsn=0/6DB4540, redo lsn=0/6DB4508
2023-10-15 21:03:56.116 PKT [2716] ERROR:  update or delete on table "transactions" violates foreign key constraint "customer_statements_tran_id_fkey" on table "customer_statements"
2023-10-15 21:03:56.116 PKT [2716] DETAIL:  Key (tran_id)=(20) is still referenced from table "customer_statements".
2023-10-15 21:03:56.116 PKT [2716] STATEMENT:  DELETE FROM public.transactions
	    WHERE tran_id IN
	        (20);
2023-10-15 21:07:30.029 PKT [3412] LOG:  checkpoint starting: time
2023-10-15 21:07:30.922 PKT [3412] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.851 s, sync=0.021 s, total=0.893 s; sync files=9, longest=0.014 s, average=0.003 s; distance=12 kB, estimate=12 kB; lsn=0/6DB7678, redo lsn=0/6DB7640
2023-10-15 22:52:51.933 PKT [2668] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-10-15 22:52:51.933 PKT [2716] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-10-15 22:52:51.933 PKT [11824] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-10-15 22:52:51.933 PKT [1268] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-10-15 22:52:51.933 PKT [12468] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
